src/__tests__/utils/setupTests.ts

```diff
@@ -75,6 +75,8 @@ jest.mock('taskcluster-client-web', () => {
   };
 });
 
+jest.mock('json-e', () => ({ jsone: jest.fn() }));
```

Comment:
I'm curious, why is it necessary? I would have prefered that we don't mock this library if that's possible.

Reply:
![Screenshot 2024-07-30 at 2 17 06 PM](https://github.com/user-attachments/assets/0a7d02fb-8899-416c-9f1e-b0d49f386a77)
I have this error that I was trying to fix. I still don't have a solution yet.